src/current/_includes/v23.1/ui/statements-table.md

```diff
@@ -10,18 +10,24 @@ Statements | SQL statement [fingerprint](#sql-statement-fingerprints). To view a
 Retries | Cumulative number of automatic (internal) [retries]({{ link_prefix }}transactions.html#transaction-retries) by CockroachDB of statements with this fingerprint within the time interval.
-% of All Runtime | How much time this statement fingerprint took to execute compared to all other statements that were executed within the time period. It is expressed as a percentage. The runtime is the mean execution latency multiplied by the execution count.
-Regions/Nodes | The regions and nodes on which statements with this fingerprint executed. <br><br>Nodes are not visible for {{ site.data.products.serverless }} clusters.
-Diagnostics | Activate and download [diagnostics](#diagnostics) for this fingerprint. To activate, click the **Activate** button. The [Activate statement diagnostics](#activate-diagnostics-collection-and-download-bundles) dialog displays. After you complete the dialog, the column displays the status of diagnostics collection (**WAITING**, **READY**, or **ERROR**). Click <img src="{{ 'images/common/ui-ellipsis-button.png' | relative_url }}" alt="Vertical ellipsis" /> and select a bundle to download or select **Cancel request** to cancel diagnostics bundle collection. <br><br>Statements are periodically cleared from the Statements page based on the start time. To access the full history of diagnostics for the fingerprint, see the [Diagnostics](#diagnostics) tab of the Statement Details page. <br><br>Diagnostics is not visible for {{ site.data.products.serverless }} clusters.
+Regions/Nodes | The regions and nodes on which statements with this fingerprint executed. <br><br>Nodes are not visible for {{ site.data.products.serverless }} clusters or for clusters that are not multi-region.
 Last Execution Time (UTC)| The timestamp when the statement was last executed.
 Statement Fingerprint ID | The ID of the statement fingerprint.
+Diagnostics | Activate and download [diagnostics](#diagnostics) for this fingerprint. To activate, click the **Activate** button. The [Activate statement diagnostics](#activate-diagnostics-collection-and-download-bundles) dialog displays. After you complete the dialog, the column displays the status of diagnostics collection (**WAITING**, **READY**, or **ERROR**). Click <img src="{{ 'images/common/ui-ellipsis-button.png' | relative_url }}" alt="Vertical ellipsis" /> and select a bundle to download or select **Cancel request** to cancel diagnostics bundle collection. <br><br>Statements are periodically cleared from the Statements page based on the start time. To access the full history of diagnostics for the fingerprint, see the [Diagnostics](#diagnostics) tab of the Statement Details page. <br><br>Diagnostics is not visible for {{ site.data.products.serverless }} clusters.
```

Comment:
This seems like a lot of info to put into a table cell. Maybe it makes sense for the description to keep the first sentence and create a subheading in the Statements table section that outlines the steps. WDYT?

Reply:
Kept in table cell, because information is related to the cell and a Diagnostics section already exists for the Diagnostics tab. It would be confusing to have 2 Diagnostics sections.